website/content/docs/configuration/target-aliases/create-target-alias.mdx

```diff
@@ -0,0 +1,538 @@
+
+<Tabs>
+<Tab heading="UI" group="ui">
+
+1. Log in to Boundary.
+1. Select **Targets** in the navigation pane.
+1. Select the target you want to add an alias to.
+1. Under the **Aliases** heading in the right sidebar, click **Add an alias**.
+1. Complete the following fields:
+   - **Name**: (Optional) Enter an optional name for the alias to use for identification purposes.
+   - **Description**: (Optional) Enter an optional description for the alias to use for identification purposes.
+   - **Type**: Select **Target**.
+   At this time, targets are the only Boundary resources that supports aliasing.
+   - **Alias Value**: Enter the alias value you want to use in commands to represent the target.
+   An alias's value can be a hostname or a DNS-like string.
+   - **Target ID**: This field contains the ID of the target you selected to add an alias to.
+   It is read only.
+   - **Host ID**: (Optional) Enter an optional host ID, if you would like to specify that the alias always uses the same host when you use it to connect to a target.
+1. Click **Save**.
+
+</Tab>
+<Tab heading="CLI" group="cli">
+
+1. Log in to Boundary.
+1. Use the following command to create an alias:
+
+   ```shell-session
+   $ boundary aliases update target \
+       -destination-id tcp_1234567890 \
+       -id alt_1234567890 \
+       -authorize-session-host-id hst_1234567890
+   ```
+
+   You can use any of the following [attributes](/boundary/docs/concepts/domain-model/aliases) when you update an alias:
+
+   - `-description=<string>` - Specifies the optional description you want to use for identification purposes.
+   - `-destination-id=<string>` - Specifies the ID of the target that the alias references.
+   - `id=<string>` - Specifies the ID of the alias you want to update.
+   - `-name=<string>` - Specifies the optional name you want to use to describe the alias for identification purposes.
+   - `-scope-id=<string>` - Scope in which to create the alias. The default is `global`.
+   You can also specify the scope using the BOUNDARY_SCOPE_ID environment variable.
+   At this time, aliases are only supported for the global scope.
+   - `-value=<string>` - Specifies the string that you want to use as the alias to represent the target.
+   The alias `value` must comply with DNS naming rules.
+   - `-authorize-session-host-id=<string>` - Optionally indicates the host ID to use when you use the alias to authorize a session.
+
+</Tab>
+</Tabs>
+
+## Create multiple aliases for a single target
+
+Target aliases point directly to the target they are associated with. Targets can be assigned a host set or a direct target address.
+
+[Host sets](/boundary/docs/concepts/domain-model/host-sets) are sets of functionally equivalent hosts, and are commonly used for deployments at scale. A target assigned a host set will select a host from the host set at random when Boundary establishes a connection.
```

Comment:
Technically the host is selected when Boundary authorizes a session, not when a connection is made. This is an important distinction, since a session can have multiple connections, but every connection in that session will connect to the same host.